src/addons/dragAndDrop/index.js

```diff
@@ -75,44 +125,26 @@ export default function withDragAndDrop(
         this.state.isDragging && 'rbc-addons-dnd-is-dragging'
       )
 
+      let EventComponent = components.event || TitleComponent
+      if (resizable) {
+        EventComponent = withEventResizability(
```

Comment:
this neat, one issue here tho is that this creates a new class per render, which is not cheap, for a lot of reasons. THe big one being React can never reuse the the markup because the render -> render the "type" is actually different

Reply:
I wouldn't use a HOC here, instead I'd move the `withEventResizability` logic directly into the event wrapper component, or if there is value in splitting it out (does anything else use the HOC?) Use it on the wrapper component wrapping `props.children` instead, leaving EventComponent alone until it's being rendered. The other thing that avoids, btw, is the fact that a user can specify an EventComponent _per_ view like:

```
components = {
  event: DefaultEventComponent,
  month: { event: MonthEventComponent },
  week: {  event: WeekEventComponent },
}
```

Reply:
I just noticed you said: 

> implemented all of the drag & resize in the EventWrapper - no more withEventResizability 

above, Sorry ignore those bits of the comments :)